src/components/axes/axes.ts

```diff
@@ -245,8 +263,12 @@ export class AxesComponent extends SignalWatcher(ChromeProvider(LitElement)) {
     yValues: ReadonlyArray<Hertz>,
     canvasSize: Readonly<Size>,
   ) {
-    const xTitleFontSize = this.calculateFontSize(this.xTitle);
-    const yTitleFontSize = this.calculateFontSize(this.yTitle);
+    this.cachedXTitleSize ??= this.calculateFontSize(this.xTitle);
+    this.cachedYTitleSize ??= this.calculateFontSize(this.yTitle);
+
+    const xTitleFontSize = this.cachedXTitleSize;
+    const yTitleFontSize = this.cachedYTitleSize;
+
```

Comment:
at the minimum the comment is now out of date,


but the comment's suggestion is good. Why don't we do that?